test/metabase/api/user_test.clj

```diff
@@ -229,9 +230,10 @@
       (is (= "You don't have permissions to do that."
              (mt/user-http-request :rasta :get 403 "user", :status "all"))))
 
-    (testing "Pagination gets the total users _in query_"
-      (is (= (t2/count User)
-             ((mt/user-http-request :crowberto :get 200 "user" :status "all") :total))))
+    (testing "Pagination gets the total users _in query_, not including the Internal User"
+      (let [f (if (t2/exists? User config/internal-mb-user-id) dec identity)] ;; is there a smarter way to do this?
```

Comment:
are there cases where the internal user won't exist in tests?

if so, maybe just modify the `t2/count` query to not include the internal user ID, rather than doing it after the call gets executed?